<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(250, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="home_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(139, 139, 139);
border-bottom-color: rgb(238, 238, 238);
}

QPushButton: hover{
color: rgb(0, 0, 0);
background-color: rgb(139, 139, 139);
border-bottom-color: rgb(238, 238, 238);
}
</string>
    </property>
    <property name="text">
     <string>HOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="data_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(139, 139, 139);
border-bottom-color: rgb(238, 238, 238);
}

QPushButton: hover{
color: rgb(0, 0, 0);
background-color: rgb(139, 139, 139);
border-bottom-color: rgb(238, 238, 238);
}
</string>
    </property>
    <property name="text">
     <string>DATA CODING</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(139, 139, 139);
border-bottom-color: rgb(238, 238, 238);
}

QPushButton: hover{
color: rgb(0, 0, 0);
background-color: rgb(139, 139, 139);
border-bottom-color: rgb(238, 238, 238);
}
</string>
    </property>
    <property name="text">
     <string>TEST</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dataca_btn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(0, 0, 0);
background-color: rgb(139, 139, 139);
border-bottom-color: rgb(238, 238, 238);
}

QPushButton: hover{
color: rgb(0, 0, 0);
background-color: rgb(139, 139, 139);
border-bottom-color: rgb(238, 238, 238);
}
</string>
    </property>
    <property name="text">
     <string>DATA CALIBERATION</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>411</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;MT Extra&quot;;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>       WELCOME TO WATER QUALITY APP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>571</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images.jpeg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
